run-tests.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 TEST_TYPE="unit"
-PYTHON_BIN="python3"
+PYTHON_BIN="/usr/bin/python3"
```

Comment:
We install python3 in `/opt` in vagrant machines, and this won't be the correct path